fairscale/optim/oss.py

```diff
@@ -148,7 +148,7 @@ def load_state_dict(self, state_dict: Dict[str, Any]) -> None:
         self.load_local_state_dict(state_dict["state"][self.rank])
 
         # Restore the global param_groups
-        self.param_groups = state_dict["param_groups"]
+        self.param_groups = recursive_copy_to_device(state_dict["param_groups"], non_blocking=True, device=self._device)
```

Comment:
Line too long?

Is there a test for this already? If so, should there be some assert in the test?

Reply:
no test, it's a good point, I'll add that

Reply:
We follow PyTorch coding style of 120 columns:

https://github.com/pytorch/pytorch/blob/master/.flake8#L3

It is enforced here:

https://github.com/facebookresearch/fairscale/blob/master/setup.cfg#L29

Reply:
it's supposed to be automatic with vscode+black, it's pure FB tooling and 120 lines, this formatting crap is so annoying.. I'll fix that in the next PR

Reply:
ah no ok, black is happy with that indeed, it's 121 cols but for some reason this passes (same for the init above, also 121). Anyway, all good then